README.md

```diff
@@ -99,8 +99,8 @@ Install this project with:
 
 ```sh
   cd learderboard
-  yarn install
-  yarn dev 
+  yarn install 
+  yarn start
```

Comment:
- [ ] [OPTIONAl] Good job styling the app, as it is just one small fix I am adding it as optional but please fix this, please notice that we also have to make sure our documentation is correct, I think for running the app the` yarn start` command should be replaced with `yarn dev` instead, currently `yarn start` just starts the webpack in the watch mode :point_right:  `webpack --config webpack.config.js --watch`, running `yarn dev` is the command that starts the app in local server :+1: